### PR TITLE
[Enhancement](profile)  enhance column predicates display on profile

### PR DESCRIPTION
**Pull request by BiteTheDDDDt**
# Proposed changes

1. fix duplicate ColumnPredicates profile display
图片

2. supporte display delete condition
图片

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [ ] No
    - [ ] I don't know
4. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
5. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
6. Does it need to update dependencies:
    - [ ] Yes
    - [ ] No
7. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [ ] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 34.89 seconds
 load time: 444 seconds
 storage size: 17123343161 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20221129103843_clickbench_pr_54950.html

File: be/src/olap/column_predicate.h [hunk 1]
```diff
@@ -166,13 +166,13 @@ class ColumnPredicate {
     }
     uint32_t column_id() const { return _column_id; }
 
-    virtual std::string debug_string() {
+    virtual std::string debug_string() const {
         return _debug_string() + ", column_id=" + std::to_string(_column_id) +
```
Comment (Gabriel39) [inline be/src/olap/column_predicate.h:170]: I think `column_id` doesn't make sense for users. It's better to get the real column name according to `SegmentIterator` 's variable `_schema`

Comment (yiguolei) [reply be/src/olap/column_predicate.h:170]: column name maybe removed in the future....

**Review by Gabriel39 — APPROVED** [review]
LGTM

**Pull request merged**